Formula/gst-python.rb

```diff
@@ -10,21 +10,50 @@ class GstPython < Formula
     sha256 "e708aa675d5744fc36608249728290d6d97803cd071bf2ac91d245a9d8e8c996" => :mavericks
   end
 
+  option "without-python"
```

Comment:
Should probably be `option "without-python3"` too if I'm reading the below changes correctly?